.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: php
+php:
+  - '5.5'
+  - '5.6'
+  - '7.0'
```

Comment:
Yep, was still in the process of adding it (sadly Travis has no way to test configs without committing them to the repo), should work now.